src/pages/costModels/costModel/addSourceWizard.tsx

```diff
@@ -83,7 +86,12 @@ class AddSourceWizardBase extends React.Component<Props, AddSourcesStepState> {
         actions={[
           <Button
             key="save"
-            isDisabled={isUpdateInProgress || this.props.isLoadingSources || this.props.fetchingSourcesError !== null}
+            isDisabled={
+              this.checkChecked(this.state.checked, this.state.initialChecked) ||
```

Comment:
Please rename `checkedChecked` to something more like `!hasSelectedItems` or `!hasSelections`.

Reply:
I changed this to compareSelected as that what the function is doing.